### DISCUSSION TITLE
The server doesn't responds login

### DISCUSSION THREAD

**Discussion by cahyo47wibowo** [Shashank02051997/QuizzieThunder-Flutter#3] [General]
I tried to login with valid creadentials and fake crendentials on app but the server doesn't response, can you help please?

**Comment by Shashank02051997** [comment]
Hi @cahyo47wibowo
Firstly, thank you for taking the time to check the QuizzeThunder Flutter repo. You can use the credentials I shared with you last time to access the app.

Phone number: 6785432111
Password: sandeep

If, by chance, the provided credentials do not work, please follow these steps: 

download the QuizzieThunder-NodeJs(https://github.com/Shashank02051997/QuizzieThunder-NodeJs) repository and set up the database as per the given instructions mentioned in the readme file. Afterward, download the QuizzieThunder-Flutter repository and map the frontend flutter code to your local/main server according to the instructions provided.
